src/globus_sdk/services/timer/data.py

```diff
@@ -1,31 +1,226 @@
 from __future__ import annotations
 
-import datetime
+# the name "datetime" is used in this module, so use an alternative name
+# in order to avoid name shadowing
+import datetime as dt
 import logging
 import typing as t
 
 from globus_sdk.config import get_service_url
+from globus_sdk.exc import warn_deprecated
 from globus_sdk.services.transfer import TransferData
-from globus_sdk.utils import PayloadWrapper, slash_join
+from globus_sdk.utils import MISSING, MissingType, PayloadWrapper, slash_join
 
 log = logging.getLogger(__name__)
 
 
+class TransferTimer(PayloadWrapper):
+    """
+    A helper for specifying the payload for Transfer Timer creation.
+    Use this along with :meth:`create_timer <globus_sdk.TimerClient.create_timer>` to
+    create a timer.
+
+    .. note::
+        ``TimerClient`` has two methods for creating timers, ``create_timer`` and
+        ``create_job``.
+        ``create_job`` uses a different API -- only ``create_timer`` will work with
+        this helper class.
+
+        Users are strongly recommended to use ``create_timer`` and this helper for
+        timer creation.
+
+    :param name: A name to identify this timer
+    :type name: str, optional
+    :param schedule: The schedule on which the timer runs
+    :type schedule: dict
+    :param body: A transfer payload for the timer to use. If it includes
+        ``submission_id`` or ``skip_activation_check``, these parameters will be
+        removed, as they are not supported in timers.
+    :type body: dict or :class:`~.TransferData`
+
+    The Schedule field encodes data which determines when the Timer will run.
+    Timers may be "run once" or "recurring", and "recurring" timers may specify an end
+    date or a number of executions after which the timer will stop. A ``schedule`` is
+    specified as a dict, but the SDK provides two useful helpers for constructing these
+    data.
+    Example schedules:
```

Comment:
```suggestion

    **Example Schedules**
```

...following pattern established elsewhere

Reply:
Yep yep yep! Bold text rather than a simple normal-weight line with a colon at the end. It stands out better.